proto/palomachain/paloma/evm/turnstone.proto

```diff
@@ -38,6 +38,17 @@ message UploadSmartContract {
   uint32 retries = 5;
 }
 
+message UploadUserSmartContract {
+  bytes bytecode = 1;
+  string abi = 2;
+  bytes constructorInput = 3;
+
+  string val_address = 4 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
```

Comment:
`author` might have been a more intuitive name here.

Reply:
I started with `owner` but didn't like. Changed it to `author` now. Thanks :+1: